bin/localstack/mk-s3

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
+pushd $(git rev-parse --show-toplevel)
 
 # find repo root name (currently civiform), also support any names for backward and forward compatibility
-CIVIFORM_DIR=$(pwd | awk -F / '{print $(NF-2)}')
+CIVIFORM_DIR=${PWD##*/}
```

Comment:
Suggestion (non-blocking): this is the same as `$(basename $PWD)`

Reply:
done